docs/src/markdownDocs/react.mdx

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import { Divider } from '../layout/Divider';
+import { CodeBlock } from '../shared';
+
+{/* KEYWORDS: react web framework guide tutorial */}
+
+# Getting Started with React
+
+The following instructions will guide you through starting a new Brightlayer UI project using React. Alternatively, you can also start your project by downloading one of our code samples from [Github](https://github.com/etn-ccis?q=blui) (the Login or Navigation examples are good projects to start from).
+
+## Using the Brightlayer UI CLI
+
+The fastest way to start a new Brightlayer UI project is to use the [Brightlayer UI CLI](/development/cli). The CLI will automatically create a starter project for you (using [Create React App](https://github.com/facebookincubator/create-react-app)) with all of the necessary Brightlayer UI dependencies pre-configured. To start a React project:
+
+<CodeBlock sx={{ height: 'auto', fontSize: '14px' }} code={`npx -p @brightlayer-ui/cli blui new react`} />
```

Comment:
what happens when you specify the code's `language` prop as `sh` or `tsx`/`jsx`? Will it break anything?

Reply:
dayum ok

Reply:
I can take another look after you merge your PR. This is almost good to go